editor/src/clj/editor/model_loader.clj

```diff
@@ -68,7 +73,7 @@
     (let [ext (string/lower-case (resource/ext resource))]
       (if (= "dae" ext)
         (load-collada-scene stream)
-        (load-model-scene stream (resource/path resource))))))
+        (load-model-scene resource stream (resource/path resource))))))
```

Comment:
We can get the path from the resource, so let's get rid of the path argument here.